Makefile

```diff
@@ -67,16 +67,11 @@ htex_local_alternate_test: ## run all tests with htex_local config
 $(WORKQUEUE_INSTALL):
 	parsl/executors/workqueue/install-workqueue.sh
 
-
-work_queue_procs := $(shell ps aux | grep -E -e "[0-9]+:[0-9]+ work_queue_worker" | tr -s ' ' | cut -f 2 -d " ")
-work_queue_killcmd := $(if $(work_queue_procs), "kill" "-3" $(procs), "echo" "no work_queue_workers to running")
-
 .PHONY: workqueue_ex_test
 workqueue_ex_test: $(WORKQUEUE_INSTALL)  ## run all tests with workqueue_ex config
 	@$(work_queue_killcmd)
 	work_queue_worker localhost 9000  &> /dev/null &
 	PYTHONPATH=.:/tmp/cctools/lib/python3.5/site-packages  pytest parsl -k "not cleannet" --config parsl/tests/configs/workqueue_ex.py --cov=parsl --cov-append --cov-report= --random-order --bodge-dfk-per-test
```

Comment:
I was checking the logs in travis test, and just noticed that it still manually launch `work_queue_worker`. I think it may be better to remove
```
@$(work_queue_killcmd)
work_queue_worker localhost 9000  &> /dev/null &
```
and set LocalProvider in `parsl/tests/configs/workqueue_ex.py`.

Reply:
The LocalProvider is set by default if no provider is specified.  Do you think we have to make it explicit in `parsl/tests/configs/workqueue_ex.py`?

Reply:
Also, commit 44c150a removes the standalone worker from the Makefile.

Reply:
> The LocalProvider is set by default if no provider is specified. Do you think we have to make it explicit in `parsl/tests/configs/workqueue_ex.py`?

Ah, no.